tests/snapshots/src/main/generated/index-semanticdb/locals

```diff
@@ -17,41 +17,41 @@ public class Example {
-      │                      ╰──┬─────┬───┬╯                  │
-      │                         │     │  ^│                   │
-      │                         v     │  ││                   │
-      │      ╭────────────────────╮   │  ││                   │
-      │      │definitionResult(46)│   │  ││                   │
-      │      ╰────────────┬───────╯   │  ││                   │
-      │                   │           │  ││                   │
-      │                   │           ╰──┼┼──────────────╮    │
-      │                   │   ╭──────────╯│              │    │
-      ╰───────────────╮   │   │           │              │    │
-         ╭────────────┼───┼───┼───────────╯              │    │
-         │            │   │   │   ╭──────────────────────┼────╯
-         │            │   │   │   │                      │
-         v            v   v   │   v                      v
- ╭──────────────╮ ╭───────────┴──────╮ ╭───────────────────────────────────╮
- │moniker local0│ │range(50) 3:34 'n'│ │hoverResult(54) ```java\nint n\n```│
- ╰──────────────╯ ╰──────────────────╯ ╰───────────────────────────────────╯
+       ╰─────┬────────────┬──────────╯       ╰────┬───────────┬──╯
+             │            │                     ^ │           │
+      ╭──────╯            │                     │ │           │
+      │                   │     ╭───────────────┼─╯           │
+      │                   │     │               │             │
+      │                   v     v               │             │
+      │            ╭──────────────────╮         │             │
+      │            │range(56) 3:46 'n'│         │             │
+      │            ╰────────────┬─────╯         │             │
+      │                         │    ╭──────────╯             │
+      │                         v    │                        │
+      │                    ╭─────────┴───╮                    │
+      │                    │resultSet(43)│                    │
+      │                    ╰──┬──┬──┬────╯                    │
+      │                       │  │  │  ^                      │
+      │ ╭─────────────────────┼──╯  │  │                      │
+      │ │                     v     │  │                      │
+      │ │    ╭────────────────────╮ │  │                      │
+      │ │    │definitionResult(46)│ │  │                      │
+      │ │    ╰──────────┬─────────╯ │  │                      │
+      │ │               │   ╭───────┼──╯                      │
+      │ │               │   │       ╰──────────────────╮      │
+      ╰─┼───────────╮   │   │                          │      │
+        │           │   │   │   ╭──────────────────────┼──────╯
+        │           │   │   │   │                      │
+        v           v   v   │   v                      v
+ ╭────────────╮ ╭───────────┴──────╮ ╭───────────────────────────────────╮
+ │local local0│ │range(50) 3:34 'n'│ │hoverResult(54) ```java\nint n\n```│
```

Comment:
You can ignore the rest of the diff in this file, the most interesting bit is `local local0` to indicate we now use the `local` kind for local symbols